cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleService.java

```diff
@@ -345,6 +345,16 @@ public ApplicationDetail getLatestAppDetail(ApplicationId appId) throws Exceptio
                                                                             changeSummary));
   }
 
+  /**
+   *
+   * @param namespaceId
+   * @param appId
+   * @return latest app version
+   */
+  public String getLatestAppVersion(NamespaceId namespaceId, String appId) {
```

Comment:
nit : appName

Reply:
this is a good function to have in AppLifecycleService - since we would reference  this in multiple apis to fetch the latest version